graphql_compiler/schema_generation/sqlalchemy/scalar_type_mapper.py

```diff
@@ -94,8 +96,50 @@
     mssqltypes.XML,
 })
 
-SQL_CLASS_TO_GRAPHQL_TYPE = merge_non_overlapping_dicts(
-    GENERIC_SQL_CLASS_TO_GRAPHQL_TYPE, MSSQL_CLASS_TO_GRAPHQL_TYPE)
+
+POSTGRES_CLASS_TO_GRAPHQL_TYPES = {
+    postgrestypes.UUID: GraphQLID,
+    postgrestypes.DOUBLE_PRECISION: GraphQLFloat,
+    postgrestypes.TIMESTAMP: GraphQLDateTime,
+    postgrestypes.ENUM: GraphQLString,
+}
+
+UNSUPPORTED_POSTGRES_TYPES = frozenset({
+    # We shouldn't map the Postgresql bit type to the GraphQLBoolean type. The Postgresql bit type
+    # is used to represent a bit string of variable length.
+    # https://www.postgresql.org/docs/8.1/datatype-bit.html
+    postgrestypes.TIME,
+    postgrestypes.BIT,
```

Comment:
nit: Consider moving the `BIT` type such that it's right under the comment that specifically references it